packages/forma-36-react-components/src/components/Asset/AssetIcon/README.mdx

```diff
@@ -0,0 +1,19 @@
+import { AssetIcon } from './AssetIcon';
```

Comment:
@denkristoffer awesome! I think README could be based on the following structure:

1. Description
2. Variations/Overview
3. Examples of usage
4. Best practices
5. Content recommendations (if applying)
6. Accessibility (if applying)

This is what we were trying to do in other components, like Button or Accordion. I will try to document it somehow, maybe create template for it or something.

Reply:
I tried to follow this structure now, although I don't really know about best practices for these components. Do you think there's anything to say there?